docs/kb/api-version-4/prospects.md

```diff
@@ -351,7 +375,10 @@ Requests that attempt to subscribe a prospect to lists that it is already subscr
 
 To modify a prospect's matching status for associated profile criteria, the Pardot ID of the profile criteria is required. Once the ID is obtained, an `update` is submitted with parameters formatted as `profile_criteria_<profile_criteria_id>=<status>`. The value of `<status>` may be either `match`, `nomatch`, or `unknown`.
 
-_**Example:** Setting a profile criteria for a prospect whose email address is_ `bob@pardot.com` _to_ `match`: /api/prospect/version/4/do/update/email/[bob@pardot.com](mailto:bob@pardot.com)?profile_criteria_8=match&amp;api_key=&amp;user_key=_**Example:** Setting a profile criteria for a prospect with Pardot ID_ `58` _to_ `nomatch`: /api/prospect/version/4/do/update/id/58?profile_criteria_8=nomatch&amp;api_key=&amp;user_key=
+_**Example:** Setting a profile criteria for a prospect whose Pardot ID is_ `58` _to_ `match`: /api/prospect/version/4/do/update/id/58?profile_criteria_8=match&amp;api_key=&amp;user_key=
+
+_**Example:** Setting a profile criteria for a prospect with Pardot ID_ `58` _to_ `nomatch`: /api/prospect/version/4/do/update/id/58?profile_criteria_8=nomatch&amp;api_key=&amp;user_key=
```

Comment:
ID is_ `58` for this one or ID_ `58` for the one above, either way but be consistent